tlslite/extensions.py

```diff
@@ -529,6 +529,71 @@ def parse(self, p):
 
         return self
 
+
+class SupportedVersionsExtension(TLSExtension):
+    """
+    This class handles the SupportedVersion extensions used in TLS 1.3.
+
+    See draft-ietf-tls-tls13.
```

Comment:
all those draft references will be replaced with the RFC number after it is published, it's temporary, and I don't expect to publish a "release" quality version before TLS 1.3 is finalised anyway.